src/main/resources/api.yaml

```diff
@@ -520,6 +579,24 @@ components:
             - $ref: '#/components/schemas/NoteTypeDto'
           example: "board"
       required: [ title, description, workspaceId, type ]
+    NoteUpdateDto:
```

Comment:
I am not sure if we should add another DTOs for updating the resources (we could just use the actual resource DTOs like NoteDto and QuizDto) - you do not have to manually check the given values in UpdateDto, just that fields which cannot change stays the same (f.e. quiz id). Personally I think such solution would be better

Reply:
I would feel a bit weird using objects which should be half empty. If you analize it from code perspective you see that you get object which has i.e. 6 fields. How can you know that 3 are always null and it's ok? It may cause some problems imo